clusterloader2/testing/network/1_1_ratio_override.yaml

```diff
@@ -0,0 +1,2 @@
+Ratio: "1:1"
```

Comment:
I missed that during the original review - can you please capitalize those?

We're using the convention that all env-vars that we're passing are capitalized, e.g.:
https://github.com/kubernetes/perf-tests/blob/master/clusterloader2/testing/experimental/storage/pod-startup/cluster_load_scale_by_nodes/override.yaml

Reply:
Same for all variables

Reply:
yes i have capitalized the variables.